File: Iniciante/URI 2786.hs
main :: IO ()
main = do
    l' <- getLine
    c' <- getLine

    let l = read l' :: Int
    let c = read c' :: Int
    let lajes1 = (l * c) + ((l - 1) * (c - 1)) :: Int
    let lajes2 = ((l - 1) * 2) + ((c - 1) * 2) :: Int

    putStrLn $ show lajes1
    putStrLn $ show lajes2

File: Iniciante/URI 2786.clj
(def L (read-string (read-line)))
(def C (read-string (read-line)))

(def lajes1 (+ (* L C) (* (- L 1) (- C 1))))
(def lajes2 (+ (* (- L 1) 2) (* (- C 1) 2)))

(println lajes1)
(println lajes2)